setup-signify.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail 
+
+sudo apt -y install signify-openbsd qrencode
+
+# clear out old keys
+rm -f /vx/config/key.pub /vx/config/key.sec
+
+# Generate the keypair
+signify-openbsd -G -n -p /vx/config/key.pub -s /vx/config/key.sec
```

Comment:
I'm unsure what the permissions should be for `/vx/config` to be able to run this without `sudo`.

Reply:
There's no reason we couldn't run with sudo I don't think, @benadida is that right?

Reply:
vx-admin can write to /Vx/config, and that's the user that will generate the keys.

Reply:
so no need for sudo then?